spring/spring-data/src/test/java/io/quarkus/ts/spring/data/di/ListInjectionIT.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.ts.spring.data.di;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@Tag("QUARKUS-2095")
+@QuarkusScenario
+public class ListInjectionIT {
+
+    @QuarkusApplication(classes = ListInjectionBean.class)
+    static RestService app = new RestService()
+            .withProperty(ListInjectionBean.LIST_INJECTION_BEAN_ENABLED, ListInjectionBean.LIST_INJECTION_BEAN_ENABLED_VALUE);
```

Comment:
```suggestion
    static RestService app = new RestService().withProperty(LIST_INJECTION_BEAN_ENABLED, LIST_INJECTION_BEAN_ENABLED_VALUE);
```
looks better, use static imports